nbviewer/providers/github/handlers.py

```diff
@@ -35,21 +35,30 @@
 
 from .. import _load_handler_from_location
 
-
 def _github_url():
     return os.environ.get('GITHUB_URL') if os.environ.get('GITHUB_URL', '') else "https://github.com/"
 
 class GithubClientMixin(object):
+    """
+    provider_label: str
+        Text to to apply to the navbar icon linking to the provider
+    provider_icon: str
+        CSS classname to apply to the navbar icon linking to the provider
+    executor_label: str, optional
+        Text to apply to the navbar icon linking to the execution service
+    executor_icon: str, optional
+        CSS classname to apply to the navbar icon linking to the execution service
+    """
```

Comment:
Are these instance attributes? Constructor arguments? Let's add a note to the beginning of the docstring explaining what these are.